src/utils/trusts.ts

```diff
@@ -102,8 +103,12 @@ const containsTrustData = (trusts: Trust[]): boolean => {
  * @param appData Application Data in Session
  * @param trustId Trust ID find (returns empty object if not found)
  */
-const getTrustByIdFromApp = (appData: ApplicationData, trustId: string): Trust => {
-  return appData[TrustKey]?.find(trust => trust.trust_id === trustId) ?? {} as Trust;
+const getTrustByIdFromApp = (appData: ApplicationData, trustId: string, isReview?: boolean): Trust => {
+  if (isReview) {
```

Comment:
Following on from the comment in the `details.mapper`, these changes can be discarded